src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/MarkTaskReconfigurationServiceTest.java

```diff
@@ -186,6 +191,34 @@ void should_not_mark_tasks_to_reconfigure_if_task_resource_cannot_be_reconfigura
         assertEquals(0, taskResourcesMarked);
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    void should_fail_is_reconfigurable_when_task_attributes_are_empty() {
```

Comment:
is this testing failure when task attributes are empty?

Reply:
Hi Ben, Thank you pointing it. :) The test should fail when the taskAttributes are null. I should rename this appropriately.